modules/security/src/testFixtures/java/org/apache/ignite/internal/configuration/stub/StubAuthenticationProviderListView.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.configuration.stub;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.ignite.configuration.NamedListView;
+import org.apache.ignite.internal.configuration.AuthenticationProviderView;
+import org.jetbrains.annotations.Nullable;
+
+/** Stub of {@link NamedListView} for tests. */
+public class StubAuthenticationProviderListView implements NamedListView<AuthenticationProviderView> {
```

Comment:
Can these classes be replaced with `InjectConfiguration`?

Reply:
Not sure. I've tried it. Validators haven't been called on injected configuration.